zone/client_packet.cpp

```diff
@@ -904,9 +904,9 @@ void Client::CompleteConnect()
 	heroforge_wearchange_timer.Start(250);
 
 	// enforce some rules..
-	if (!CanBeInZone()) {
+	if (!CanEnterZone()) {
 		LogInfo("Kicking character [{}] from zone, not allowed here (missing requirements)", GetCleanName());
-		GoToSafeCoords(ZoneID("arena"), 0);
+		GoToSafeCoords(RuleI(World, FailedRequirementBootZoneID), 0);
```

Comment:
It's possible that the client still can't even enter what's configured for a default here.

It might make sense to have it reroute them to bind